addons/website_blog/controllers/main.py

```diff
@@ -120,8 +121,8 @@ def _prepare_blog_values(self, blogs, blog=False, date_begin=False, date_end=Fal
             'date_begin': date_begin,
             'date_end': date_end,
             'first_post': first_post.with_prefetch(post_ids),
-            'other_tags': other_tags,
-            'tag_category': tag_category,
+            'other_tags': other_tags if all_tags else [],
```

Comment:
wrong idea to have conditionally RecordSet and List in the qcontext !

You should return a recordSet empty in this special case in all_tags... mapped and filtered will just become a noop.
And don't hesitate to write the condition in 2 lines when it become unreadable.

Else about this fix, a backport of commit fa0b098900573f37740cb41141307fee03fb5533 should be enough...